### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ https://github.com/webcomponents/custom-elements-manifest/issues/68
 - Added CustomElementField that extends ClassField with the ability to
   describe associated attributes. Fixes https://github.com/webcomponents/custom-elements-manifest/issues/36
 
+- Added a `"deprecated"` field to FunctionLike objects (functions and methods) ([#105](https://github.com/webcomponents/custom-elements-manifest/pull/105))
```
Comment (kevinpschaaf) [inline CHANGELOG.md:31]: Also add summary to `returrn` from https://github.com/webcomponents/custom-elements-manifest/pull/109?

**Review by kevinpschaaf — APPROVED** [review]
From a diff of the last released version, looks like these were also missed:

* Add `deprecated` to all interfaces (not just FunctionLike)
* Add `CustomElementMixinDeclaration`
* Make `CustomElementDeclaration` extend `CustomElement` also

File: CHANGELOG.md (hunk 1 above)
Comment (justinfagnani) [reply CHANGELOG.md]: That's included in #109

**Comment by justinfagnani** [comment]
CustomElementMixinDeclaration was already included in the changelog as a fix.

**Pull request merged**